### DISCUSSION TITLE
CLI Feature Suggestion - chia wallet get_last_transaction

### DISCUSSION THREAD

**Discussion by toastygunn** [Chia-Network/chia-blockchain#6573] [Ideas]
Use-Case: sometimes I want to know the last transaction timestamp from a script by querying my chia wallet daemon.
Currently, you either have to know the transactionID (I generally dont), search the log file, OR page through all of the 'chia wallet get_transactions' until you get to the end.

Being able to query and get the details on last transaction or last two transactions would be nice.

**Comment by d1Mm** [comment]
`chia wallet get_transactions -o -2` prints the last two at the top of the list
